ui/admin/tests/acceptance/host-catalogs/host-sets/selectors.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export const FIELD_PREFERRED_ENDPOINT = '[name="preferred_endpoints"] input';
+export const ADD_PREFERRED_ENDPOINT_BUTTON =
```

Comment:
none blocking comment: very nit pick comment, but can we change `_BUTTON` by `_BTN` [for consistency with other button selectors](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/helpers/selectors.js#L6-L8), what do you think?